src/govuk/components/radios/template.njk

```diff
@@ -56,11 +56,12 @@
         {%- if item.divider %}
         <div class="govuk-radios__divider">{{ item.divider }}</div>
         {%- else %}
+        {% set isChecked = item.checked or (params.value and item.value == params.value) %}
```

Comment:
I'd recommend checking `item.checked` for undefined, not truthiness. If it's explicitly false you want that stored and it not to fall to the other conditions.

Reply:
In js that would be the equivalent of the [Nullish coalescing operator (??)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) but in Nunjucks I *think* this could be:
```suggestion
        {% set isChecked = (item.checked if (item.checked !== undefined) else (params.value and item.value == params.value) %}
```